src/test/xml_coverage.go

```diff
@@ -38,14 +42,108 @@ func parseXMLLines(lines []xmlCoverageLine) []core.LineCoverage {
 	return ret
 }
 
+// Covert the Coverage result to XML bytes, ready to be write to file
+func CoverageResultToXML(sources []core.BuildLabel, coverage core.TestCoverage) []byte {
+	linesValid := 0
+	linesCovered := 0
+	validFiles := coverage.OrderedFiles()
+
+	// get the string representative of sources
```

Comment:
are you sure this is correct? I would have thought the sources element in the XML would represent the actual file paths?

Reply:
Yes, that's usually the case when running coverage. However I thought we could have it as a please target, like so `test/python_rules/pytest:all`? Since we are running it through please, so it would be easy to tell what we run to cover. However I could make it represent the path like `test/python_rules/pytest`, and leaving out the `all` bit  if you'd like?